app/views/users/sessions/new.html.erb

```diff
@@ -1,17 +1,16 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l">Log in</h2>
-
+  <h1 class="govuk-heading-m">Sign in with your email address</h1>
     <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
       <div class='govuk-form-group'>
-        <%= f.govuk_email_field :email, autocomplete: "email", value: @email %>
+        <%= f.govuk_email_field :email, label: { text: '' }, autocomplete: "email", value: @email %>
```

Comment:
It's probably not very accessible to have empty labels.

I would add some text, could even be 'email' and set `hidden` to true.

```
label: {text: "Emain", hidden: true},
```

Reply:
I'm not sure that will do what you want: https://www.w3.org/WAI/tutorials/forms/labels/#note-on-hiding-elements

Reply:
Hehe, but we are simply telling the experienced technology at our formbuilder to visually hide it this way.

Really what it does is applies a class `.govuk-visually-hidden` to the label, and that class is 1px by 1px in an unassuming place :)

Reply:
Or so I think anyway, judging by how hidden labels on https://govuk-form-builder.netlify.app/building-blocks/fieldsets/ are handled.

Reply:
That's good, but surprising. I would have expected a `visually_hidden` attribute or similar 🙂